components/WrapperCanvas.vue

```diff
@@ -24,23 +33,23 @@
 
     function deleteElement(eId) {
         // delete last element of map
-        props.panel.deleteElement(eId);
+        props.comic.getPage(0).getStrip(0).panels[props.panelIndex].deleteElement(eId);
     }
 
     function resizeElement(obj) {
         // validate element id
         validateElementId(obj.eId);
         // update element width and height
-        elements.get(obj.eId).setPos({ x: obj.pos.x, y: obj.pos.y });
-        elements.get(obj.eId).setWidth(obj.width);
-        elements.get(obj.eId).setHeight(obj.height);
+        elements.get(obj.eId).pos = { x: obj.pos.x / panelDimensions.width, y: obj.pos.y / panelDimensions.height };
+        elements.get(obj.eId).setWidth(obj.width / panelDimensions.width);
+        elements.get(obj.eId).setHeight(obj.height / panelDimensions.height);
```

Comment:
I would think that it is nice to have functions in the Element that go like: setXForWidth(xValue, width) and the same for getting. To reduce errors while setting and getting.